applications/SwimmingDEMApplication/swimming_DEM_application.cpp

```diff
@@ -96,6 +97,7 @@ void KratosSwimmingDEMApplication::Register()
   KRATOS_REGISTER_3D_VARIABLE_WITH_COMPONENTS(VECTORIAL_ERROR_1)
   KRATOS_REGISTER_3D_VARIABLE_WITH_COMPONENTS(DISPLACEMENT_OLD)
   KRATOS_REGISTER_3D_VARIABLE_WITH_COMPONENTS(AVERAGED_FLUID_VELOCITY)
+  KRATOS_REGISTER_VARIABLE(MASS_SOURCE)
```

Comment:
Not sure if this `MASS_SOURCE` variable creation is a leftover from the previous implementation.